### PR TITLE
update release notes for 6.0.0 release

### PR DESCRIPTION
**Pull request by elfenheart**

### CODE REVIEW

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 1]
```diff
@@ -30,6 +30,140 @@ Cask Data Application Platform Release Notes
    :backlinks: none
    :depth: 2
 
+`Release 6.0.0 <http://docs.cask.co/cdap/6.0.0/index.html>`__
+=============================================================
+
+New Features
+------------
+
+- :cask-issue:`CDAP-14330` - Added Google Cloud Storage copy and move action plugins
```
Comment (hadarhg) [inline cdap-docs/reference-manual/source/release-notes.rst]: missing period

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 2]
```diff
@@ -30,6 +30,140 @@ Cask Data Application Platform Release Notes
...
+
+- :cask-issue:`CDAP-14533` - New pipeline list user interface.
+
+- :cask-issue:`CDAP-14613` - Added minimap to pipeline canvas.
```
Comment (hadarhg) [inline cdap-docs/reference-manual/source/release-notes.rst:59]: code tags around `minimap`?

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 3]
```diff
@@ -30,6 +30,140 @@ Cask Data Application Platform Release Notes
...
+
+- :cask-issue:`CDAP-14645` - Added support for running CDAP system services in Kubernetes environment.
+
+- :cask-issue:`CDAP-14657` - Add the ability to copy and paste a node in pipeline studio.
```
Comment (hadarhg) [inline cdap-docs/reference-manual/source/release-notes.rst]: Let's keep it consistent and always use "Added"
So, change "Add" to "Added" here and wherever else needed.

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 4]
```diff
@@ -30,6 +30,140 @@ Cask Data Application Platform Release Notes
...
+
+- :cask-issue:`CDAP-14617` - Significantly improve performance of Transactional Messaging System.
+
+- :cask-issue:`CDAP-14821` - Added early validation for the properties of the Google BigQuery sink to fail during pipeline deployment instead of at runtime.
+
+- :cask-issue:`CDAP-14823` - Improved the error message when a null value is read for a non-nullable field in avro file sources.
+
+- :cask-issue:`CDAP-15047` - Improved loading of system artifacts to load in parallel instead of sequentially.
+
+- :cask-issue:`CDAP-15059` - Improve Google Cloud Dataproc provisioner to allow configuring default projectID from CDAP configuration.
```
Comment (hadarhg) [inline cdap-docs/reference-manual/source/release-notes.rst]: Improve --> Improved

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 5]
```diff
@@ -30,6 +30,140 @@ Cask Data Application Platform Release Notes
...
+Bug Fixes
+---------
+
+- :cask-issue:`CDAP-12211` - Fixed a casting bug for the DB source where unsigned integer column were incorrectly being treated as integers instead of longs.
+
+- :cask-issue:`CDAP-13410` - Removed the need for ZooKeeper for service discovery in remote runtime environment.
+
+- :cask-issue:`CDAP-7230` - Fixed an issue with recording lineage for realtime sources.
+
+- :cask-issue:`CDAP-12941` - Fixed dynamic spark plugin to use appropriate context classloader for loading dynamic spark code.
```
Comment (hadarhg) [inline cdap-docs/reference-manual/source/release-notes.rst]: Capitalize Spark in both places in the sentence.

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 6]
```diff
@@ -30,6 +30,140 @@ Cask Data Application Platform Release Notes
...
+
+- :cask-issue:`CDAP-13554` - Fixed a bug that caused mapreduce pipelines to fail when using too many macros.
```
Comment (hadarhg) [inline cdap-docs/reference-manual/source/release-notes.rst]: MapReduce

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 7]
```diff
@@ -30,6 +30,140 @@ Cask Data Application Platform Release Notes
...
+
+- :cask-issue:`CDAP-14691` - Fixed a bug that would cause workspace ids to clash when wrangling items of the same name.
+
+- :cask-issue:`CDAP-14702` - Fixed a bug in secure store caused by breaking changes in Java update 171. Users should be able to get secure keys on java 8u171.
+
+- :cask-issue:`CDAP-14708` - Fixed a bug that caused Google Cloud Dataproc clusters to fail provisioning if a firewall rule that denies ingress traffic existed in the project.
+
+- :cask-issue:`CDAP-14709` - Fixed a bug that would cause data preparation to fail when preparing a large file in Google Cloud Storage.
+
+- :cask-issue:`CDAP-14724` - Fixed a bug that caused action only pipelines to fail when running using a cloud profile.
```
Comment (hadarhg) [inline cdap-docs/reference-manual/source/release-notes.rst]: action-only

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 8]
```diff
@@ -30,6 +30,140 @@ Cask Data Application Platform Release Notes
...
+
+- :cask-issue:`CDAP-14744` - Fixed an issue with adding business tags to an entity.
+
+- :cask-issue:`CDAP-14778` - Fixed an issue in handling metadata search parameters.
+
+- :cask-issue:`CDAP-14779` - Fixed a bug that would cause pipelines to fail on remote clusters if the very first pipeline run was an action only pipeline.
```
Comment (hadarhg) [inline cdap-docs/reference-manual/source/release-notes.rst]: action-only

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 9]
```diff
@@ -30,6 +30,140 @@ Cask Data Application Platform Release Notes
...
+
+- :cask-issue:`CDAP-14951` - Fixed a bug in the Google BigQuery sink that would cause pipelines to fail when writing to a dataset in a different region.
+
+- :cask-issue:`CDAP-15001` - Fixed a race condition in processing profile assignments.
+
+- :cask-issue:`CDAP-15013` - Fixed an issue that could cause inconsistencies in metadata.
+
+- :cask-issue:`CDAP-15069` - Fixed an issue with displaying workspace metadata in the UI.
+
+- :cask-issue:`CDAP-15127` - Fixed a race condition in the remote runtime scp implementation that can causing process to hang.
```
Comment (hadarhg) [inline cdap-docs/reference-manual/source/release-notes.rst]: can --> could

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 10]
```diff
@@ -30,6 +30,140 @@ Cask Data Application Platform Release Notes
...
+
+- :cask-issue:`CDAP-15196` - Fixed an issue with metadata search result pagination.
+
+- :cask-issue:`CDAP-15223` - Fixed Wrangler DB connection where a bad JDBC driver can stay in cache for 60 minutes, making DB connection not usable.
```
Comment (hadarhg) [inline cdap-docs/reference-manual/source/release-notes.rst]: can --> could

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 11]
```diff
@@ -30,6 +30,140 @@ Cask Data Application Platform Release Notes
...
+
+- :cask-issue:`CDAP-15249` - Fixed a NullPointerException in Google Cloud Dataproc provision when there is no network being configured.
```
Comment (hadarhg) [inline cdap-docs/reference-manual/source/release-notes.rst]: "..provision for when there was no network configured."

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 12]
```diff
@@ -30,6 +30,140 @@ Cask Data Application Platform Release Notes
...
+
+- :cask-issue:`CDAP-15299` - Fixed a bug that caused some aggregator and joiner keys to be dropped if they hashed to the same value as another key.
+
+- :cask-issue:`CDAP-15332` - Fixed a bug in the RuntimeMonitor that doesn't reconnect through SSH correctly, causing failure in monitoring the correct program state.
```
Comment (hadarhg) [inline cdap-docs/reference-manual/source/release-notes.rst:161]: code tags needed around `RuntimeMonitor`?

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 13]
```diff
@@ -30,6 +30,140 @@ Cask Data Application Platform Release Notes
...
+
+- :cask-issue:`CDAP-15369` - Fix Google Cloud Dataproc runtime for Google Cloud Platform projects where OS Login is enabled.
+
+
+Deprecated and Removed Features
+-------------------------------
+
+- :cask-issue:`CDAP-15241` - Deprecated plugins, HDFSMove and HDFSDelete plugins are removed from core plugins.
```
Comment (hadarhg) [inline cdap-docs/reference-manual/source/release-notes.rst]: Can you clarify this sentence? Not sure I understood the meaning, but if the following is correct, then how about: 
Removed HDFSMove and HDFSDelete plugins from core plugins.
(Wasn't sure if to use "Removed" here or "Deprecated".)

**Review by CuriousVini — APPROVED** [review]

**Pull request merged**